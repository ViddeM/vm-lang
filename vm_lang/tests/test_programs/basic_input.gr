fn main() {
    let a_bool = read_bool();
    print_bool(!a_bool);

    let a_string = read_string();
    print_string(a_string);

    let a_number = read_number();
    print_number(a_number * 2);
}