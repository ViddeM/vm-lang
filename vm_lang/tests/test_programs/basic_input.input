true
testing the coolest of things
42