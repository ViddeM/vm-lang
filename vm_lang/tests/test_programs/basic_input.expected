false
testing the coolest of things
84